dali/operators/reader/sequence_reader_op.cc

```diff
@@ -55,8 +55,8 @@ Example directory structure::
       ....
 
 .. note::
-  This operator is an analogue of VideoReader working on video frames extracted as separate images.
-  It's main purpose is for test baseline. For regular usage, the VideoReader is
+  This operator is an analogue of video reader working on video frames extracted as separate images.
+  It's main purpose is for test baseline. For regular usage, the video reader is
```

Comment:
I know it's not yours, but 
```suggestion
  Its main purpose is for test baseline. For regular usage, the video reader is
```